ddpui/core/orgtaskfunctions.py

```
@@ -195,14 +199,55 @@
             try:
                 prefect_service.delete_deployment_by_id(dataflow.deployment_id)
             except Exception:
+                # we want to return an error if the deployment exists in prefect
+                # but failed to be deleted
+                # we want to ignore it if the deployment doesn't exist
+                # hmmm
                 pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)